db/migrate/20170710163523_create_resources_textsearch.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 # This migration has been ported over from Conjur v3's /authz project. I had to 
-# make a few changes for possum that have been documented with comments.
+# make a few changes for Conjur v5 that have been documented with comments.
```

Comment:
WDYT about calling this Conjur OSS instead? It's confusing here because the Conjur version here is currently `1.4.7`, so it's not obvious where 5 comes from.

Reply:
Yeah sounds good. You're right v5 is not really meaningful these days.